CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 3.2.0
+July 24th, 2019
```

Comment:
Change the date to tomorrow July 25th. I want 1 more day to test it out manually myself 😅